### PR TITLE
Dap2 bad request error

### PR DESCRIPTION
**Pull request by dh-opendap**
Changes to HDF5_Handler to throw BESSyntaxUserError when variables are determined to be incompatible with requested encoding response.  That is, when CFEnable=false and a DAP2 response is requested, and the DDS contains incompatible types which for its present incarnation this handler only identifies int64 types as incompatible

### CODE REVIEW

File: modules/hdf5_handler/h5get.cc [hunk 1]
```diff
@@ -2546,3 +2552,14 @@ string handle_string_special_characters_in_path(const string &instr) {
     return outstr;
 }
 
+string invalid_type_error_msg(
+        string var_type
+){
+    stringstream msg;
+
+    msg << "Your request was for a (DAP2 data model response). ";
```
Comment (jgallagher59701) [inline modules/hdf5_handler/h5get.cc:2560]: Let's change this -> "Your request was for a (DAP2 data model response). "
To  "Your request was for a response that uses the DAP2 data mode. "

**Review by jgallagher59701 — APPROVED** [review]
Make that change and merge. Thanks.

File: modules/hdf5_handler/h5get.cc (hunk 1 above)
Comment (dh-opendap) [reply modules/hdf5_handler/h5get.cc]: Good suggestion.  Made it '... response that uses the DAP2 data model.'

**Comment by jgallagher59701** [comment]
I'm going to merge this.

**Pull request merged**